UPGRADE.md

```diff
@@ -2,6 +2,42 @@
 
 ## dev-develop
 
+### New definition mechanism for image-formats
+
+A new structure for the image-formats configuration files was introduced.
+For an explanation on how to define image-formats in the new way please
+refer to the documentation (TODO: link to documentation).
+
+Out of the box, image-formats defined in the old way do still work,
+but the XMLLoader and commands are marked as deprecated.
+However when using more profound functionality regarding the image-formats,
+there are some BC breaks:
+
+#### `sulu_media.image_format_file` changed to `sulu_media.image_format_files`
+The configuration `sulu_media.image_format_file` of the MediaBundle
+was changed to `sulu_media.image_format_files` and the type was changed
+from a scalar to an array.
+
+#### "Command" renamed to "Transformation"
+Internally the concept of a command on an image was renamed to 
+"transformation". This renaming was consequently executed throughout the
+MediaBundle. This BC break is only important when custom commands have
+been created. To update the custom commands (now transformations) they now have
+to implement the `TransformationInterface` instead of the `CommandInterface`.
+Moreover the service tag under which a transformation gets registered changed
+from `sulu_media.image.command` to `sulu_media.image.transformation`. Needless
```

Comment:
Remove `Needless to say`, it's not really clear and only bloats the sentence. Seems also a little bit arrogant.